utils/context.go

```diff
@@ -0,0 +1,31 @@
+package utils
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+const MetadataKey = "viam-metadata"
+
+func ContextWithMetadata(ctx context.Context) (context.Context, map[string][]string) {
+	md := make(map[string][]string)
```

Comment:
```suggestion
	md := map[string][]string{}
```

Reply:
Happy to make this change, but any reason this syntax is preferred over `make`?